test/test_bson.py

```diff
@@ -788,6 +788,11 @@ def test_regex_from_native(self):
     def test_regex_hash(self):
         self.assertRaises(TypeError, hash, Regex('hello'))
 
+    def test_regex_comparison(self):
+        re1 = Regex('a')
+        re2 = Regex('b')
+        self.assertNotEqual(re1, re2)
```

Comment:
Can you add some more assertions that make sure flags are compared properly too? And an `self.assertEqual` with two equivalent `Regex`'s?

Reply:
Of course